exporter/collector/metrics.go

```diff
@@ -245,7 +245,20 @@ func (me *MetricsExporter) PushMetrics(ctx context.Context, m pmetric.Metrics) e
 			s, _ := status.FromError(err)
 			st = statusCodeToString(s)
 
-			recordPointCountDataPoint(ctx, len(ts), st)
+			succeededPoints := len(ts)
+			failedPoints := 0
+			for _, detail := range s.Details() {
+				if summary, ok := detail.(*monitoringpb.CreateTimeSeriesSummary); ok {
+					failedPoints = int(summary.TotalPointCount - summary.SuccessPointCount)
+					succeededPoints = int(summary.SuccessPointCount)
+				}
+			}
+
+			// always record the number of successful points
```

Comment:
Does recording 0 do anything (e.g. cause a new metric to show up/ not show up)?  My instinct is that we should always record both successful points and failed points, but i'm not sure what the impact is.

Reply:
The problem is that there are many different failure codes, so we could send `0` but we would need to pick a failure code to go with it. Otherwise in the case of 0 failures we'll be sending "`X succeeded and 0 succeeded`" if we just used the code we parse from the success response

Reply:
ah, got it.  This works then.